#! /bin/sh -e
## 10_export_vfs.dpatch by Daniel Kobras <kobras@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Export VFS functions.
## DP: Patch pulled from upstream BK tree.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -Nru a/libaudiofile/audiofile.exports b/libaudiofile/audiofile.exports
--- a/libaudiofile/audiofile.exports	2004-06-14 03:32:51 -07:00
+++ b/libaudiofile/audiofile.exports	2004-06-14 03:32:51 -07:00
@@ -102,3 +102,12 @@
 afTellFrame
 afWriteFrames
 afWriteMisc
+af_fclose
+af_flength
+af_fread
+af_fseek
+af_ftell
+af_fwrite
+af_virtual_file_destroy
+af_virtual_file_new
+af_virtual_file_new_for_file
